src/common/types/trustedCertificateFields.ts

```diff
@@ -0,0 +1,6 @@
+import { InfraEnv } from '../api';
+
+export type TrustedCertificateFieldsType = {
```

Comment:
Do we need a separate file for this type?

I suggest we rename the file `proxyFields` file to `GenerateIsoFields` (for example) and add the required types here.

So we can define `DiscoveryImageFormValues` there too.